src/data/finset/lattice.lean

```diff
@@ -236,14 +236,32 @@ end
 
 end sup
 
-lemma disjoint_sup_right [distrib_lattice α] [order_bot α] {a : α} {s : finset β} {f : β → α} :
-  disjoint a (s.sup f) ↔ ∀ i ∈ s, disjoint a (f i) :=
+section distrib_lattice
+variables [distrib_lattice α] [order_bot α] {s : finset β} {f : β → α} {a : α}
+
+lemma disjoint_sup_right : disjoint a (s.sup f) ↔ ∀ i ∈ s, disjoint a (f i) :=
```

Comment:
This lemma's here twice now right?

Reply:
Yup, merge accident. I'm gonna PR them separately anyway.

Reply:
Haven't I already?

Reply:
My mistake!